### ISSUE TITLE
How to use medium-editor with modal bootstrap

### ISSUE THREAD

**Issue by vominhtam** [yabwe/medium-editor#1360]
Hi all,

I have a issue when using with modal boostrap. When I click all button (B,I) all event not working
I don't know how to fix it. please help me?

Thank all so musch

**Comment by paltman** [comment]
same here

**Comment by leotiger** [comment]
@vominhtam, @paltman.

I had the same problem. Please check several things of your modal configuration, there are several incompatible settings:

- data-backdrop set to static
- data-keyboard set to false
- tabindex set to "-1"

To solve the different issues I removed the three attributes from the modal tag. 

Moreover I had to add a special instruction to fix the link popup:

.medium-editor-toolbar-form-active {
    visibility: visible;
}

You may configure this dynamically via javascript, if you're modal does not contain a medium editor instance you may activate these settings or vice-versa.

Related issues: #284, #1222 and more. Just search for modal in the issues and you'll find several hints.

**Comment by nchase** [comment]
@vominhtam @paltman does leotiger's suggestion resolve your issue?

**Comment by stepofweb** [comment]
One more issue on bootstrap modal (or any ajax method, I guess).

1. Open ajax modal - ok, working as expected.
2. close the modal
3. open the modal again (or another ajax modal using medium-editor) - not working until I refresh the page.

I use destroy() and reinit medium-editor using a different unique ID... but it's not working.
So... destroy() is not really destroying!

**Comment by marcosignacio** [comment]
Yep, removing those 3 parameters and adding that css rule fixes this issue.

Thanks!

**Comment by dbachet** [comment]
Hi, just for the record, I tried many rich text editors using `contenteditable=true` and every of them wouldn't work properly when it comes to action buttons. I tried medium-editor lastly and moving it outside the modal worked properly. 

So I assume that the modal issue might be linked with the `contenteditable` thing but I might be wrong.

**Comment by srknzl** [comment]
How to remove that property effectively? @dbachet

**Comment by rluetke** [comment]
Find my solution with changing `elementsContainer` in #1222
